Vagrantfile

```diff
@@ -68,27 +68,12 @@ module Forklift
         @host_vars = {box['name'] => box['ansible']['variables']}
 
         if (playbooks = box['ansible']['playbook'])
-
-          if playbooks.is_a?(String)
-            args = SUPPORT_NAMED_PROVISIONERS ? ['main', type: 'ansible'] : [:ansible]
```

Comment:
oh, i assumed since it wasn't supported by both paths that it was maybe just something leftover? alright i'll see if i can squeeze it back in there.